examples/data_augmentation.py

```diff
@@ -12,6 +12,9 @@
 import torch.nn as nn
 import torch.optim as optim
 from torch.utils.data import Dataset, DataLoader
+import torchvision
+from matplotlib import pyplot as plt
+import cv2
```

Comment:
have you checked that the examples are well generated ?

Reply:
I recovered the previous file and with the help of @dvd42 we will generate an example for Data augmentation with all the modules